docs/versioning.md

```diff
@@ -56,3 +55,32 @@ It MAY include patch level changes. Patch version MUST be reset to 0 when minor
 Major version **X** MUST be incremented for a release that introduces backward incompatible changes.
 A major release can also include minor and patch level changes.
 You must reset the patch and minor version to 0 when you change the major version.
+
+## MFTF version table
+
+This table lists the version of the MFTF that was released with a particular version of Magento.
+
+|||
+|---|---|
+| Magento | MFTF |
+| 2.3.1 | 2.3.13 |
+| 2.3.0 | 2.3.9 |
+| 2.2.8 | 2.3.13 |
+| 2.2.7 | 2.3.8 |
+
+## Updating MFTF
```

Comment:
There is a topic about MFTF update: https://devdocs.magento.com/mftf/docs/update.html
Shouldn't it go there?
By the way, the topic uses `composer update` to update dependencies. (This should be reviewed I believe because this command updates all Magento dependencies, not only MFTF).

Reply:
Removed.